packages/@react-spectrum/tooltip/test/Tooltip.test.js

```diff
@@ -40,4 +40,15 @@ describe('Tooltip', function () {
     let tooltip = getByRole('tooltip');
     expect(ref.current.UNSAFE_getDOMNode()).toBe(tooltip);
   });
+
+  it('click does not propagate to parent', () => {
+    let mockClick = jest.fn();
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions
+    let {getByRole} = render(<div onClick={mockClick}><Tooltip>This is a tooltip</Tooltip></div>);
+    let tooltip = getByRole('tooltip');
+    fireEvent.pointerDown(tooltip);
+    fireEvent.click(tooltip);
+    fireEvent.mouseDown(tooltip);
```

Comment:
Technically this is the wrong order of events compared to the browser. Lets simplify and use our util function to handle this.
```suggestion
    triggerPress(tooltip);
```